docs/nydus-failover-upgrade.md

```diff
@@ -37,6 +39,22 @@ Example request body:
 
 By now only the field `nydusd_path` is required in the request body. More fields (like `version`, `policy`, etc) may be used in the future.
 
+#### By modifying the config.toml file and restarting nydus-snapshotter
+
+A hot upgrade is triggered by changing the nydusd_path in the nydus-snapshotter configuration file. When nydus-snapshotter restarts, it compares the expected nydusd version with the currently running version and automatically starts the hot upgrade process if they are inconsistent.
+
+Here's an example of how to modify the nydusd_path in config.toml file to point to a new binary:
+
+```toml
+# Old configuration
+# nydusd_path = "/usr/local/bin/old-nydusd"
+
+# New configuration
+nydusd_path = "/usr/local/bin/new-nydusd"
```

Comment:
Hi @Zephyrcf, could users simply update the binary and restart snapshotter to trigger the hot upgrade, instead of modifying the path in the config?